ligo-webide-frontend/ligo-components/packages/eth-project/src/Project/index.js

```diff
@@ -23,6 +23,14 @@ modelSessionManager.registerModeDetector(filePath => {
       return 'settings'
     } else if (base.endsWith('.sol')) {
       return 'solidity'
+    } else if (base.endsWith('.religo')) {
+      return 'javascript'
```

Comment:
shouldn't this be reasonligo?

Reply:
No, as there is no `reasonligo` in monaco editor and in https://ide.ligolang.org/ they also use `javascript` for `religo`

Reply:
ok, sounds good